src/components/formatters/test/CFormatterJsonBase_test.cc

```diff
@@ -166,15 +182,14 @@ TEST(CFormatterJsonBaseTest, JSonArrayValueToSmartObj_ExpectSuccessful) {
   // Arrange value
   const char* json_array =
       "[\"test1\", \"test2\", \"test3\"]";  // Array in json format
-  Json::Value json_value;  // Json value from array. Will be initialized later
+
+  utils::json::JsonValue::ParseResult json_value_parse = utils::json::JsonValue::Parse(json_array);
```

Comment:
@OHerasym I suggest that adding of `using utils::json::JsonValue` and `using utils::json::JsonValueRef`to the beginning of file would be good.